PowerEditor/src/Notepad_plus.cpp

```diff
@@ -601,6 +601,12 @@ LRESULT Notepad_plus::init(HWND hwnd)
 	//Search menu
 	//disable "Search Results Window" under Search Menu 
 	::EnableMenuItem(_mainMenuHandle, IDM_FOCUS_ON_FOUND_RESULTS, MF_DISABLED | MF_GRAYED | MF_BYCOMMAND);
+	::EnableMenuItem(_mainMenuHandle, IDM_SEARCH_GOTONEXTFOUND, MF_DISABLED | MF_GRAYED | MF_BYCOMMAND);
+	::EnableMenuItem(_mainMenuHandle, IDM_SEARCH_GOTOPREVFOUND, MF_DISABLED | MF_GRAYED | MF_BYCOMMAND);
+
+	// Find dialog is not created yet
+	::EnableMenuItem(_mainMenuHandle, IDM_SEARCH_FINDNEXT, MF_BYCOMMAND | MF_DISABLED | MF_GRAYED);
+	::EnableMenuItem(_mainMenuHandle, IDM_SEARCH_FINDPREV, MF_BYCOMMAND | MF_DISABLED | MF_GRAYED);

```

Comment:
Can be removed now.

Reply:
It does nothing if find dialog is not created
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/ea71d5ea4e2116fa30d1ab0dda6b1cb2c160b8de/PowerEditor/src/NppCommands.cpp#L1362

And It makes no sense that IDM_SEARCH_GOTONEXTFOUND and  DM_SEARCH_GOTOPREVFOUND are enabled if search results panel was not created.